src/routes/e2e/widget/inline/+page.svelte

```diff
@@ -19,9 +20,23 @@
   }
 
   form.onMount((component) => console.log(component));
-  // TODO: Use a more specific type
-  journey.onSuccess((response) => (userResponse = response?.user));
-  journey.onFailure((response) => console.log(response.journey?.error));
+
+  journey.onSuccess((response) => {
+    userResponse = response?.user;
+
+    if (codeParam || suspendedIdParam) {
+      history.replaceState(null, '', '/e2e/widget/modal');
+    }
+  });
+
+  journey.onFailure((response) => {
+    console.log('Singleton onFailure event fired');
+    console.log(response?.journey?.error);
+
+    if (codeParam || suspendedIdParam) {
+      history.replaceState(null, '', '/e2e/widget/modal');
+    }
+  });
```

Comment:
All of this code just allows the full end-to-end flow for Social Login.

Reply:
So by full end-to-end flow do you mean the e2e test will take us as far as getting redirected to the social provider, then whoever is testing enters credentials there, and the test resumes when we are returned to the app? I think this is what you are saying but want to be sure.

Reply:
Yeah, I'm not referring to automation. I'm just referring to providing the capability to manually test the Social Login flow using our e2e test pages. The automation itself still just tests for the redirect itself.